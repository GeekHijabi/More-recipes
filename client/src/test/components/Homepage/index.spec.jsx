import React from 'react';
import expect from 'expect';
// import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, configure } from 'enzyme';

import HomePage from '../../../js/components/Home/index';

// configure({ adapter: new Adapter() });

describe('Homepage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
