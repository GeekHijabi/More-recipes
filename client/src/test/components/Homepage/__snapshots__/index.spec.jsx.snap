// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should render correctly 1`] = `
<div>
  <Header />
  <withRouter(Banner) />
  <Connect(TrendingRecipe) />
  <Footer />
</div>
`;
